Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
new file mode 100644
--- /dev/null	(date 1746710352885)
+++ b/.idea/misc.xml	(date 1746710352885)
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="Black">
+    <option name="sdkName" value="FODS (2)" />
+  </component>
+  <component name="ProjectRootManager" version="2" project-jdk-name="FODS (2)" project-jdk-type="Python SDK" />
+</project>
\ No newline at end of file
Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
new file mode 100644
--- /dev/null	(date 1746709876063)
+++ b/.idea/modules.xml	(date 1746709876063)
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/Group-Project-Diabetes.iml" filepath="$PROJECT_DIR$/.idea/Group-Project-Diabetes.iml" />
+    </modules>
+  </component>
+</project>
\ No newline at end of file
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
new file mode 100644
--- /dev/null	(date 1746709876150)
+++ b/.idea/vcs.xml	(date 1746709876150)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="" vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: .idea/Group-Project-Diabetes.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/Group-Project-Diabetes.iml b/.idea/Group-Project-Diabetes.iml
new file mode 100644
--- /dev/null	(date 1746709876052)
+++ b/.idea/Group-Project-Diabetes.iml	(date 1746709876052)
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="PYTHON_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="jdk" jdkName="FODS (2)" jdkType="Python SDK" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
Index: .idea/inspectionProfiles/profiles_settings.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/inspectionProfiles/profiles_settings.xml b/.idea/inspectionProfiles/profiles_settings.xml
new file mode 100644
--- /dev/null	(date 1746709876091)
+++ b/.idea/inspectionProfiles/profiles_settings.xml	(date 1746709876091)
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
+  <settings>
+    <option name="USE_PROJECT_PROFILE" value="false" />
+    <version value="1.0" />
+  </settings>
+</component>
\ No newline at end of file
Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1746709829340)
+++ b/.idea/.gitignore	(date 1746709829340)
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
Index: code.ipynb
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/code.ipynb b/code.ipynb
--- a/code.ipynb	(revision 8b140d4cecda7069dfe57c05cb9d25f0477dc33a)
+++ b/code.ipynb	(date 1746711848580)
@@ -2019,7 +2019,8 @@
     "# Train/test split\n",
     "X_train, X_test, y_train, y_test = train_test_split(X_model, y, test_size=0.2, random_state=42, stratify=y)\n",
     "\n",
-    "# Initialize the LGBMClassifier\n",
+    "# \n",
+    "# LGBMClassifier\n",
     "model = lgb.LGBMClassifier(\n",
     "    objective='multiclass',\n",
     "    num_class=3,\n",
@@ -2030,7 +2031,7 @@
     "    n_estimators=1000\n",
     ")\n",
     "\n",
-    "# Train the model with early stopping and logging\n",
+    "# Train model \n",
     "model.fit(\n",
     "    X_train, y_train,\n",
     "    eval_set=[(X_train, y_train), (X_test, y_test)],\n",
@@ -2041,7 +2042,7 @@
     "    ]\n",
     ")\n",
     "\n",
-    "# Make predictions\n",
+    "# Predictions\n",
     "y_pred = model.predict(X_test)\n",
     "y_pred_probs = model.predict_proba(X_test)\n",
     "\n",
